mirascope/core/base/structured_stream.py

```diff
@@ -262,6 +265,12 @@ async def inner_async(
                 fields_from_call_args = get_fields_from_call_args(
                     response_model, fn, args, kwargs
                 )
+                if (
```

Comment:
same comment here as `_extract`, shouldn't `stream_decorator` handle this for us?